tst/testinstall/cset.tst

```diff
@@ -42,7 +42,7 @@ true
 # test intersecting cosets
 gap> Intersection(RightCoset(Group([ (1,2,3,4,5,6,7), (5,6,7) ]),(3,6)(4,7)),
 >                 RightCoset(Group([ (1,2,3,4,5,6,8), (1,3,2,6,4,5), (1,6)(2,3)(4,5)(7,8) ]),(1,7,6,8,3,5)));
-RightCoset(Group([ (2,6,7)(3,5,4), (1,2,4)(3,6,5) ]),(1,3,7,5)(\
```

Comment:
While this isn't how GAP tests are often written, I'd suggest changing this to:

```
gap> Intersection(...) = RightCoset(..);
true
```

As that is more resiliant to these annoying types of changes.

Reply:
you don't have to do this, only if it makes it easier for you to write / update tests.

Reply:
ok, will do that.
Maybe we could parse the result of the runs and check them. I had that issue when running C++ vs GAP, because of gap formatting, I had to postprocess the output in order for the comparison to match.